support-frontend/assets/pages/supporter-plus-landing/supporterPlusLanding.tsx

```diff
@@ -149,6 +160,27 @@ export function SupporterPlusLandingPage({
 	};
 	const heading = <LandingPageHeading />;
 
+	const [supportOnceDisplay, setSupportOnceDisplay] = useState(
+		useContributionsSelector(
+			isUserInAbVariant('singleLessProminent', 'variant'),
+		),
+	);
```

Comment:
@paul-daniel-dempsey I think an edge case you'll need to account for is if a user has an initial `contributionType` of 'ONE_OFF'. This can happen in 2 scenarios: 

1) If they're referred to the page after clicking Single in the choice cards in the Epic or Banner then a query string param `?selected-contribution-type=ONE_OFF` will be appended to the URL, if this query param is present them initial the `contributionType` in our Redux store is 'ONE_OFF'.

2) If they're a returning to the page and they were on the Single tab before we'll have saved this in session storage (`selectedContributionType`) and we'll set the initial `contributionType` in our Redux store based on that value.

So, if the initial `contributionType`  is `ONE_OFF` I think we'll want to set `supportOnceDisplay` to `false`, we could make that check when we define it like so ...

```
const [supportOnceDisplay, setSupportOnceDisplay] = useState(() => contributionType === 'ONE_OFF' ? false : useContributionsSelector(
		isUserInAbVariant('singleLessProminent', 'variant'),
	));
```

You can test this locally by appending the `?selected-contribution-type=ONE_OFF` query param to the URL.

https://support.thegulocal.com/uk/contribute?selected-contribution-type=ONE_OFF#ab-singleLessProminent=variant

Reply:
Ah yes, I can see the  ?selected-contribution-type=ONE_OFF Query Param would force you onto Monthly otherwise. Not what is need. Will try to remember query parameters can change your starting point, can see there is an amountFromURL too but thats not relevant here. Thanks, changed as recommended.